app/serializers/billing-detail.js

```diff
@@ -0,0 +1,8 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  extract: function(store, type, payload, id, requestType) {
+    this.__requestType = requestType; // used by `extractMeta`
```

Comment:
@sandersonet currently it seems to work without the serializer, but I recall that I had to have it even when loading a single model since otherwise it broke horribly. There is also a test that currently fails when I remove the serializer, I'm not sure why.